archetype/src/main/resources/archetype-resources/workdir/run.sh

```diff
@@ -15,6 +15,4 @@ coordinator     --memberWorkerCount 2 \
                 --monitorPerformance \
                 test.properties
 
-provisioner --download
```

Comment:
Note (to myself): There are some more code parts where we have `provisioner --download`, e.g. in the sample scripts created by the Simulator Wizard or in the `worker.sh`